content/en/blog/_posts/2022-12-12-host-process-containers-stable/index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Windows HostProcess Containers Are Going Stable"
+date: 2022-12-13
+slug: host-process-containers-stable
+---
+
+**Authors**: Mark Rossetti (Microsoft) and Brandon Smith (Microsoft)
+
+The long-awaited day has arrived: HostProcess containers, the Windows equivalent to Linux privileged
+containers, has finally made it to **GA in Kubernetes 1.26**!
+
+Let's recap - what are HostProcess containers and why are they useful? Cluster operators
+are often faced with the need to configure their nodes upon provisioning. Whether it's
+installing services, certificates, network configs, device plugins, or even monitoring solutions like
+prometheus, HostProcess containers enable you to it all with minimal effort.
+
+You can create your own ultra-thin HostProcess container image to be run directly as
+a process on the host directly after spinning up a new node. All you need to do is define
+and test your administrative workload once, package it into a container, and have Kubernetes
+deploy the workload automatically. It removes the need to exec into the node or configure
+VMSS extensions to perform these administrative actions. You get all the benefits of
+containerized packaging and deployment methods combined with a reduction in both administrative
+and development cost.
+
+HostProcess containers differ quite significantly from regular Windows Server containers.
+They are run directly as processes on the host under the access policies of
+a user you specify. HostProcess containers run as either the built-in system accounts or
+ephemeral users within a user group defined by you. HostProcess containers also share
+the host's network namespace and access/configure storage mounts visible to the host.
+On the other hand, Windows Server containers are highly isolated and exist in a separate
+silo'd namespace. Direct access to the host from a Windows Server container is explicitly disallowed
+by default. Mounting a host volume into the Windows Server container is highly discouraged to minimize
+potential attack surfaces. HostProcess containers enable you to configure the host such that communication
+with your Windows container workloads remains within the bounds of a secure pipeline.
```

Comment:
```suggestion
by default. 

The Kubernetes project strongly recommends against mounting a host volume into a Windows Server
container in order to minimize potential attack surfaces. HostProcess containers enable you to configure 
the host, along with privileged helper Pods, so that communication with your Windows workloads
in containers happens within the bounds of a secure pipeline.
```

Reply:
After the suggested changes are committed I'll add a remark about how hostProcess containers are only allowed in the privileged pod-secrutiry standard profile and link to https://kubernetes.io/docs/concepts/security/pod-security-standards/#baseline